packages/esm-patient-medications-app/src/order-basket/order-basket-item.component.tsx

```diff
@@ -56,7 +56,8 @@ export default function OrderBasketItemTile({ orderBasketItem, onItemClick, onRe
             &mdash; {orderBasketItem.route?.value} &mdash; {orderBasketItem.frequency?.value} &mdash;{' '}
             {t('refills', 'Refills').toUpperCase()} {orderBasketItem.numRefills}{' '}
             {t('quantity', 'Quantity').toUpperCase()}{' '}
-            {`${orderBasketItem.pillsDispensed} ${orderBasketItem.quantityUnits?.value.toLowerCase()}`}
+            {`${orderBasketItem.pillsDispensed} ${orderBasketItem.quantityUnits?.value.toLowerCase()}`} &mdash; {' '}
+            {`${orderBasketItem.patientInstructions}`}
```

Comment:
If the `patientInstructions` are not available, then the `mdash` will be left hanging. Please wrap the mdash properly.
```suggestion
            {`${orderBasketItem.pillsDispensed} ${orderBasketItem.quantityUnits?.value.toLowerCase()}`} 
            {orderBasketItem.patientInstructions && <>&mdash; {orderBasketItem.patientInstructions}</>}
```

Reply:
Please wrap all the remaining mdashes properly too.
Thanks!

Reply:
> Please wrap all the remaining mdashes properly too.
> Thanks!

is this an instruction for mdashes in the entire file?